learnEcl/0200-syntax.md

```diff
@@ -77,21 +77,21 @@ SomeResult;"
 
 ## Definition
 
-Assigning an expression to an attribute. Definitions can't not be executing unless it is wrapped in an action and are defined by `:=`. Let's take a look at an example:
+Assigning an expression to an attribute. Definitions cannot be executing unless it is wrapped in an action and are defined by `:=`. Let's take a look at an example:
```

Comment:
"Definitions cannot be executing unless" -> "A definition cannot be executed unless"